src/admin/navigations/views/NavigationBarDetail.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/variables';
+
+// Elements
+.c-menu-detail__table {
```

Comment:
`.c-navigation...`

Reply:
fixed in commit on pr higher on the stack:
https://github.com/viaacode/avo2-client/pull/1683/commits/1084917d5c4b842be4ff0d47fac51a268eafa026
https://github.com/viaacode/react-admin-core-module/pull/89/commits/ae3129a4a30f123b1a4821203adca7267c366b6c